From 3f7cadb9259e079e4e23e4c72cc793aa434fb534 Mon Sep 17 00:00:00 2001
From: Andriy Grytsenko <andrej@rep.kiev.ua>
Date: Tue, 20 Nov 2012 14:26:19 +0200
Subject: [PATCH 09/11] Hotkeys for sort modes and for New Empty File

This is modification of commit by Vadim Ushakov but with another
keybindings instead of Ctrl+Fx as latter are often used by WM.
---
 src/main-win-ui.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/main-win-ui.c b/src/main-win-ui.c
index ae8c0a5..37fdf86 100644
--- a/src/main-win-ui.c
+++ b/src/main-win-ui.c
@@ -177,7 +177,7 @@ static GtkActionEntry main_win_actions[]=
     {"Reload2", NULL, NULL, "<Ctrl>R", NULL, G_CALLBACK(on_reload)},
     /* for popup menu - these entries exist in FmFolderView popup still */
     {"NewFolder", "folder", N_("Folder"), "<Ctrl><Shift>N", NULL, G_CALLBACK(bounce_action)},
-    {"NewBlank", "text-x-generic", N_("Blank File"), NULL, NULL, G_CALLBACK(bounce_action)}
+    {"NewBlank", NULL, N_("Empty File"), "<Ctrl><Alt>N", NULL, G_CALLBACK(bounce_action)}
 };
 
 /* main_win_toggle_actions+main_win_mode_actions - see 'ViewMenu' for mnemonics */
@@ -206,10 +206,10 @@ static GtkRadioActionEntry main_win_sort_type_actions[]=
 
 static GtkRadioActionEntry main_win_sort_by_actions[]=
 {
-    {"ByName", NULL, N_("By _Name"), NULL, NULL, COL_FILE_NAME},
-    {"ByMTime", NULL, N_("By _Modification Time"), NULL, NULL, COL_FILE_MTIME},
-    {"BySize", NULL, N_("By _Size"), NULL, NULL, COL_FILE_SIZE},
-    {"ByType", NULL, N_("By File _Type"), NULL, NULL, COL_FILE_DESC}
+    {"ByName", NULL, N_("By _Name"), "<Alt><Ctrl>1", NULL, COL_FILE_NAME},
+    {"ByMTime", NULL, N_("By _Modification Time"), "<Alt><Ctrl>2", NULL, COL_FILE_MTIME},
+    {"BySize", NULL, N_("By _Size"), "<Alt><Ctrl>3", NULL, COL_FILE_SIZE},
+    {"ByType", NULL, N_("By File _Type"), "<Alt><Ctrl>4", NULL, COL_FILE_DESC}
 };
 
 static GtkRadioActionEntry main_win_side_bar_mode_actions[]=
-- 
1.8.0.1

